packages/okta-react/package.json

```diff
@@ -72,6 +72,7 @@
   "jest": {
     "setupFiles": [
       "./test/jest/setup.js"
-    ]
+    ],
+    "testURL": "http://localhost/"
```

Comment:
I think this was fixed in Jest 23.5.0 https://github.com/facebook/jest/issues/6766#issuecomment-412516712, maybe let's update Jest instead